mhep/mhep/assessments/tests/test_urls.py

```diff
@@ -9,9 +9,9 @@
 
 def test_list_assessments(assessment: Assessment):
```

Comment:
Should this function name be updated to `test_list_create_assessments`?

Reply:
Nope, because it's testing the **list** capability